rust/src/write.rs

```diff
@@ -108,13 +108,17 @@ pub struct WriteOptions {
     compression: Option<Compression>,
     profile: String,
     chunk_size: Option<u64>,
+    #[cfg(feature = "zstd")]
```

Comment:
Did the library already use this feature configuration option or is it being introduced here in this PR?